Dima/netflix-roulette/src/Pages/Home/Home.tsx

```diff
@@ -87,13 +99,20 @@ class Home extends React.Component<PropsFromRouteAndRedux> {
         </div>
         <Sort
           location = { this.props.location }
+          history = {this.props.history}
           handleSearchChange = { this.handleSearchChangeForSort }
           numberOfFilms = { films.length }
         />
-        <Films 
-          films = { films }
-          // handlePagination = { this.handlePgination }
-        />
+        <InfiniteScroll
+          pageStart={ films.length }
+          loadMore={ () => this.handlePagination(films.length + 10, true) }
+          hasMore={ true }
```

Comment:
Eventually movies will end and you will get an error, so you have to write a check here. Make sure that total > this.props.movies.length;